packages/playwright/tests/chromedash-guide-new-page_pwtest.js

```diff
@@ -0,0 +1,62 @@
+// @ts-check
+import { test, expect } from '@playwright/test';
+import { captureConsoleMessages, delay, login, logout } from './test_utils';
+
+test.beforeEach(async ({page}) => {
+  captureConsoleMessages(page);
+  test.setTimeout(90000);
+
+  // Attempt to login before running each test.
+  await login(page);
+});
+
+test.afterEach(async ({page}) => {
+  await logout(page);
+});
+
+
+test('navigate to create feature page', async ({page}) => {
+  // console.log('navigate to create feature page');
+
+  // Expect create feature button to be present.
+  const createFeatureButton = page.locator('[data-testid=create-feature-button]');
+  await expect(createFeatureButton).toBeVisible({timeout: 30000});
+
+  // Take a screenshot of header with "Create feature" button.
+  await expect(page.locator('[data-testid=header]')).toHaveScreenshot('create-feature-button.png');
```

Comment:
How does this play out in the scenario where a developer makes a code change and forgets to update the golden screenshot?  What kind of error message do they get and at what point do they get it?

Reply:
If they run the pwtests locally, then they will see the failures in their console.  Failures show up in the packages/playwright/tests-results directory.  If the change that causes a failure is pushed to github, then the playwright action will fail, and an artifacts file will be created with the differences.   If some other expectation fails that is not about screenshots, then the test will fail, and they would have to look into the action log.

The `npm run pwtests-update --packages=playwright` is the command to update all the screenshots locally.  They end up in the packages/playwright/tests/__screenshots__ directory.

I think I have to correct the README about this.